### ISSUE TITLE
Add @types/umzug to dependencies or peerDependencies

### ISSUE THREAD

**Issue by localnet** [mikro-orm/mikro-orm#224]
**Describe the bug**
Seems that `@types/umzug` is a required dependency when Typescript is configured with `"strict": true`, I think it should included in mikro-orm `dependencies` or `peerDependencies`.

**Stack trace**
```
node_modules/mikro-orm/dist/migrations/MigrationStorage.d.ts(1,25): error TS7016: Could not find a declaration file for module 'umzug'. 'node_modules/umzug/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/umzug` if it exists or add a new declaration (.d.ts) file containing `declare module 'umzug';`
node_modules/mikro-orm/dist/migrations/Migrator.d.ts(1,45): error TS7016: Could not find a declaration file for module 'umzug'. 'node_modules/umzug/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/umzug` if it exists or add a new declaration (.d.ts) file containing `declare module 'umzug';
```

**To Reproduce**
1. Configure `tsconfig.json` with `"strict": true`

**Versions**

| Dependency | Version |
| - | - |
| node | 12.13.0 |
| typescript | 3.6.3 |
| mikro-orm | 3.0.0-alpha.32 |
| your-driver | mariadb |

**Comment by B4nan** [comment]
There has to be something wrong but for me it works fine (pg driver) without those typings. Peer dep wont help here, but generally typings should be dev deps only.

**Comment by localnet** [comment]
Yeah, I know and I always declare typings as `devDependencies` but this time I don't know what is happening.

This is my full tsconfig.json:
```json
{
  "compilerOptions": {
    "target": "es2019",
    "module": "commonjs",
    "lib": [
      "es2019"
    ],
    "sourceMap": true,
    "outDir": "dist/",
    "rootDir": "src/",
    "removeComments": true,
    "importHelpers": true,
    "strict": true,
    "noUnusedLocals": true,
    "noUnusedParameters": true,
    "noImplicitReturns": true,
    "noFallthroughCasesInSwitch": true,
    "moduleResolution": "node",
    "esModuleInterop": true,
    "experimentalDecorators": true,
    "emitDecoratorMetadata": true
  }
}
```

**Comment by localnet** [comment]
Maybe, are you using `skipLibCheck`? in tsconfig.json.

**Comment by B4nan** [comment]
Actually looks like i do not have the strict mode enabled in that particular project, so it might be just that.

**Comment by lookfirst** [comment]
working example of failure: https://github.com/lookfirst/mikro-example

**Comment by B4nan** [comment]
Fixed in `3.0.0-alpha.33`.
